source/download/index.html.haml

```diff
@@ -35,7 +49,7 @@ The <a href="https://fedoraproject.org/wiki/Cloud_SIG">Fedora Cloud SIG</a> offe
   %a.btn.btn-primary.btn-lg{:href => "https://wiki.centos.org/SpecialInterestGroup/Atomic/Download/"} CentOS Atomic Host
 
 %p.buttons.hidden-xs
-  %a.btn.btn-primary.btn-lg{:href => "https://getfedora.org/atomic/download/"} Fedora Atomic Host
+  %a.btn.btn-primary.btn-lg{:href => "https://dl.fedoraproject.org/pub/alt/atomic/stable/Fedora-29-updates-20190611.0/AtomicHost/"} Fedora Atomic Host
```

Comment:
would probably rather not use a link to a specific version because it will rot quick. maybe just to `https://dl.fedoraproject.org/pub/alt/atomic/stable/` instead.

Reply:
ack; wasn't sure about pointing users at a directory tree to comb through, but i'll update accordingly